pkg/metrics/interfaces.go

```diff
@@ -27,10 +27,6 @@ type CounterVec interface {
 }
 
 type GaugeVec interface {
-	// CurryWith(prometheus.Labels) (*GaugeVec, error)
-	// GetMetricWithLabelValues(lvs ...string) (prometheus.Gauge, error)
-	// MustCurryWith(prometheus.Labels) *prometheus.GaugeVec
-	// With(labels prometheus.Labels) prometheus.Counter
```

Comment:
It looks like this hunk was added in the previous commit. Please squash these commits into one.